src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -1336,66 +1293,47 @@
             az network nsg show -g MyResourceGroup -n MyNsg
         - name: Get basic information about all default NSG rules with "Allow" access.
           text: >
-            az network nsg show
-            -g MyResourceGroup
-            -n MyNsg
-            --query "defaultSecurityRules[?access=='Allow']"
+            az network nsg show -g MyResourceGroup -n MyNsg --query "defaultSecurityRules[?access=='Allow']"
 """
 
 helps['network nsg rule create'] = """
     type: command
-    short-summary: Create an NSG rule.
+    short-summary: Create a network security group rule.
+    long-summary: By default, the source address and port are '*', and the destination address is '*:80'.
```

Comment:
This isn't necessary. Default values are already displayed for each parameter.

Reply:
This is one thing that I encountered a few times: Default values and allowable values listed in the helpfiles, when they should (might?) be listed as part of the argument information in the command files. The separation of some, but not all, arguments from the helpfiles also makes them more difficult to maintain.

This isn't necessarily an argument for moving them, but if there's information on where argument help resides, I can make sure that those files are also run against any documentation linting.

Reply:
I just want to make sure they aren't listed twice. In this case, I just don't think it's necessary because if the defaults ever change, then this help text will be wrong.  In other commands, we don't describe what the default output will be with minimal parameters, so I would argue we shouldn't here either.

There are a few (rare) cases where the allowed values are shown in the help text because the parameter registration doesn't have a defined choice list. A linter rule should be able to detect this I'd think (you'd just look for repeats).

Reply:
It feels like this 'long-summary' content is more focused on making the docs website flow nice.  Maybe we sould add a `remarks` or `online` tag for content such as this.  

For in-tool use, where this is displayed RIGHT ABOVE the params showing all those defaults, this would feel pretty redundant.